tests/integration/update.bats

```diff
@@ -285,7 +331,10 @@ EOF
     [ "$status" -eq 0 ]
     check_cgroup_value "cpu.cfs_period_us" 1000000
     check_cgroup_value "cpu.cfs_quota_us" 500000
+    check_systemd_value CPUQuotaPerSecUSec 500ms
+
     check_cgroup_value "cpu.shares" 100
+    check_systemd_value CPUShares 100
```

Comment:
nit: quote everything consistently?

Reply:
yeah, I'd rather have all these useless `""` removed but yeah, we can have them everywhere.

will fix

Reply:
fixed